content/id/docs/concepts/overview/object-management-kubectl/imperative-command.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Pengelolaan Objek Kubernetes dengan Perintah Imperatif
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+Objek-objek Kubernetes bisa dibuat, diperbarui, dan dihapus secara langsung dengan menggunakan perintah-perintah imperatif yang ada pada *command-line* `kubectl`. Dokumen ini menjelaskan cara perintah-perintah tersebut diorganisir dan cara menggunakan perintah-perintah tersebut untuk mengelola objek *live*.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Kelebihan dan kekurangan
+
+Perintah `kubectl` mendukung tiga cara pengelolaan objek:
+
+* Perintah imperatif
+* Konfigurasi objek imperatif
+* Konfigurasi objek deklaratif
+
+Lihat [Pengelolaan Objek Kubernetes](/id/docs/concepts/overview/object-management-kubectl/overview/) untuk menyimak diskusi mengenai kelebihan dan kekurangan dari tiap cara pengelolaan objek.
+
+## Cara membuat objek
+
+*Tool* `kubectl` mendukung perintah-perintah berbentuk kata kerja untuk membuat beberapa tipe objek yang paling umum. Perintah-perintah tersebut diberi nama yang mudah dikenali oleh pengguna yang belum familiar dengan tipe-tipe objek Kubernetes.
+
+- `run`: Membuat sebuah objek Deployment untuk menjalankan kontainer di satu atau lebih Pod.
```

Comment:
`Deployment`, *Deployment*, or Deployment.
The same applied for others kubernetes object. wdyt @girikuncoro ?

Reply:
we will standardize this in style guide, for now Deployment is good I think (the english style guide uses this)

Reply:
ok, we'll standardize this.